components/TrackRecord.tsx

```diff
@@ -61,7 +84,7 @@ export function TrackRecord({
   if (!trackRecordUserId) return <></>
 
   return (
-    <div className="max-w-xs prose flex flex-col mx-auto w-full">
+    <div className={clsx("prose flex flex-col mx-auto w-full", className)}>
```

Comment:
The change here is to allow the width of this to vary, controlled by the parent component. On `main` currently the width of the other elements in the sidebar can vary, whereas this is fixed at 320px leading it to be centred above the other elements in some cases (see screenshot of `main` below). I think it's more natural to have the outer sidebar div controlling the width (I have now fixed it at 320px for all elements).
![Screenshot 2024-09-12 at 12 28 01](https://github.com/user-attachments/assets/850fd799-c0f7-45cf-9b0f-fde5dbc91ebe)